modules/openy_gc_auth/modules/openy_gc_auth_yusa/openy_gc_auth_yusa.routing.yml

```diff
@@ -0,0 +1,9 @@
+openy_gc_auth_yusa.account_verification:
+  path: '/vy-user/verification/{uid}/{timestamp}/{hash}'
+  defaults:
+    _controller: '\Drupal\openy_gc_auth_yusa\Controller\UserController::verifyAccount'
```

Comment:
It is equal to https://github.com/fivejars/openy_gated_content/blob/master/modules/openy_gc_auth/modules/openy_gc_auth_custom/openy_gc_auth_custom.routing.yml#L9
We have to change name for the route or use general verification approach for both items